### DISCUSSION TITLE
Quarkus releases

### DISCUSSION THREAD

**Discussion by adisnuhan** [quarkusio/quarkus#30678] [Q&A]
Hello quarkus community,

we are using maven dependency plugin to regularly check if there is a new quarkus *Final version released. This gives us chance to be alerted by Jenkins if we can upgrade the version as soon as the new version comes out. The problem what we have is that the maven plugin informs us, hey there is a new quarkus version out, but if we try to upgrade the pom.xml to new version, maven is not able to find new dependency in nexus public repository. This is kind of confusing and misleading. If we check on quarkus official page, the new version reported by maven plugin is still not visible there.

It seems that maven dependency plugin fetches new version which is still not completely uploaded by quarkus team into nexus repository.

How can we solve this issue?

Regards,
Adis

**Comment by gsmet** [comment]
Which artifact do you use to check new versions? My guess is that you're using one of the core release and the Quarkus release is done in two phases (Core first, Platform after) and the release is complete only when both have been released.

Try to track `io.quarkus.platform:quarkus-bom`, it will probably work better.
